Adds and enables some Debian and Ubuntu related webjumps which should
not be inlcuded in upstream by default.

Index: conkeror-0.9.1/modules/debian/webjumps.js
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ conkeror-0.9.1/modules/debian/webjumps.js	2010-01-13 01:31:38.000000000 +0100
@@ -0,0 +1,23 @@
+/**
+ * (C) Copyright 2008-2009 Axel Beckert
+ *
+ * Use, modification, and distribution are subject to the terms specified in the
+ * COPYING file.
+ *
+ * Webjumps to interact with Debian and Ubuntu websites
+**/
+
+require("webjump.js");
+
+// Debian //
+define_webjump("debpkg", "http://packages.debian.org/%s");
+define_webjump("debfile", "http://packages.debian.org/?searchon=contents&keywords=%s&mode=path&suite=unstable&arch=any");
+define_webjump("debbugs", "http://bugs.debian.org/%s");
+define_webjump("debpts", "http://packages.qa.debian.org/%s");
+define_webjump("debpopcon", "http://qa.debian.org/popcon.php?package=%s");
+
+// Ubuntu //
+define_webjump("ubuntupkg", "http://packages.ubuntu.com/%s");
+define_webjump("ubuntufile", "http://packages.ubuntu.com/search?searchon=contents&keywords=%s&mode=path&arch=any");
+define_webjump("ubuntubugs", "http://launchpad.net/ubuntu/+source/%s");
+define_webjump("launchpad", "https://launchpad.net/+search?field.text=%s");
Index: conkeror-0.9.1/defaults/preferences/default-modules.js
===================================================================
--- conkeror-0.9.1.orig/defaults/preferences/default-modules.js	2009-09-24 02:17:21.000000000 +0200
+++ conkeror-0.9.1/defaults/preferences/default-modules.js	2009-09-24 02:17:48.000000000 +0200
@@ -60,3 +60,6 @@
 pref("conkeror.load.page-modes/dailymotion", 1);
 pref("conkeror.load.page-modes/gmail", 0);
 pref("conkeror.load.page-modes/xkcd", 1);
+
+// Debian modules
+pref("conkeror.load.debian/webjumps", 1);
